client/components/LearnerViewDirective.js

```diff
@@ -733,6 +739,27 @@ tie.directive('learnerView', [function() {
         };
 
         /**
+          * Highlights syntax errors in the coding UI
+          *
+          * @param {number} lineNumber
+          */
+        var highlightLine = function(lineNumber) {
+          var actualLineNumber = lineNumber - 1;
+          var codeLines = document.querySelectorAll('.CodeMirror-line');
+          codeLines[actualLineNumber].className += ' tie-syntax-error-line';
+        };
+
+        /**
+          * Clears highlight from syntax errors in the coding UI
+          */
+        var clearHighlight = function() {
+          var codeLines = document.querySelectorAll('.tie-syntax-error-line');
+          for (var i = 0; i < codeLines.length; i++) {
+            codeLines[i].className = 'CodeMirror-line';
```

Comment:
Shouldn't you just remove the class you added (in case something else adds stuff to that line as well in the future)?